groovy/material/FirstDegreeMaterials.groovy

```diff
@@ -3580,5 +3580,19 @@ public class FirstDegreeMaterials {
                 .components(CobaltSulfate * 1, Water * 1)
                 .colorAverage()
                 .build()
+
+        // 8638: Co3O4
+
+        LithiumNitrate = new Material.Builder(8639, SuSyUtility.susyId('lithium_nitrate'))
+                .dust()
+                .components(Lithium * 1, Nitrogen * 1, Oxygen * 3)
+                .color(0xffffff)
+                .build()
+
+        LithiumNitrateSolution = new Material.Builder(8640, SuSyUtility.susyId('lithium_nitrate_solution'))
+                .liquid()
+                .components(LithiumNitrate * 2, Water * 1)
```

Comment:
would it be better to use a 1:1 ratio like some other solutions that're in game alr?

Reply:
Yes, but:
![image](https://github.com/user-attachments/assets/ec90e222-c939-4a14-ba6e-6b07b3b3763f)

If we make 1:1 ratio, the recipe would require additional 1000L water input - idk if it is more applicable (especially because Li2CO3 is poorly soluble in water)

Reply:
> Yes, but: ![image](https://private-user-images.githubusercontent.com/4649497/398148374-ec90e222-c939-4a14-ba6e-6b07b3b3763f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQ5NTQ2MDMsIm5iZiI6MTczNDk1NDMwMywicGF0aCI6Ii80NjQ5NDk3LzM5ODE0ODM3NC1lYzkwZTIyMi1jOTM5LTRhMTQtYmE2ZS02YjA3YjNiMzc2M2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIyMyUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMjNUMTE0NTAzWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODVjMmJkZjBkOGRkZTY2MDA0NzM2MjVmY2M2NTg2ZDI2NmM0Y2U0NDg1ODRmYWIwYjE5ODNhOThlNDgwNTEzMSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.YeM-jGSOiI_gSpqyIFOH2fvF9iAvA2p64ZyNwdm0wqs)
> 
> If we make 1:1 ratio, the recipe would require additional 1000L water input - idk if it is more applicable (especially because Li2CO3 is poorly soluble in water)

Ah yes this makes sense
I think, since this already needs anhydrous nitric acid, maybe you can just output solid LiNO3 + HNO3·H2O
LiNO3 itself tends to crystallize in the form of LiNO3·3H2O from water
ig that's why this needs anhydrous nitric acid

Reply:
We don't have (HNO3)(H2O), though it would fit here for input
In previous PRs Planet told not to bother about hydrates.
I think I'd just add water, as trainvoi asked and public consensus shifts towards it